xclim/temperature.py

```diff
@@ -73,6 +74,15 @@ def compute(*args, **kwds):
         """The function computing the indicator."""
 
 
+tn_days_below = Tasmin(identifier='tnlt_{thresh}',
+                       units='days',
+                       standard_name='number_of_days_with_air_temperature_below_threshold',
+                       long_name='Number of days with Tmin < {thresh}C',
+                       description="{freq} number of days where daily minimum temperature is below {thresh}℃",
+                       cell_methods='time: minimum within days time: sum over days',
```

Comment:
i don't think it's a case of "sum over days".

Reply:
Cell methods on the indicators become very confusing I find but  I took this based on the proposed cf standard excel sheet (from Lars Barring et al)
e.g.  Tropical nights (which is the same but simply greater than) they have:

number_of_days_with_air_temperature_above_threshold | time: minimum   within days time: sum over days | Number of   Tropical Nights (Tmin > 20C)
-- | -- | --